### PR TITLE
Raise download_maxsize

### PR DESCRIPTION
**Pull request by aguilerapy**
Raise the limit globally for download_maxsize to 4GB in settings.py #313

### CODE REVIEW

File: kingfisher_scrapy/settings.py [hunk 1]
```diff
@@ -29,6 +29,10 @@
 # See https://docs.scrapy.org/en/latest/topics/settings.html#download-delay
 # See also autothrottle settings and docs
 #DOWNLOAD_DELAY = 3
+
+# The maximum response size (in bytes) that downloader will download (default: 1073741824):
+DOWNLOAD_MAXSIZE = 4000000000
```
Comment (yolile) [inline kingfisher_scrapy/settings.py]: @aguilerapy maybe you should remove the custom config in digiwhist_poland then

File: tests/test_settings.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import kingfisher_scrapy.settings
+
+
+class TestSettings():
+    name = 'test'
+    both_name = kingfisher_scrapy.settings.BOT_NAME
+    assert both_name == kingfisher_scrapy.settings.BOT_NAME
```
Comment (yolile) [inline tests/test_settings.py]: @jpmckinney should we exclude the settings file from travis? or do you think that there is a better way to test the settings?

Comment (jpmckinney) [reply tests/test_settings.py]: Why exclude it? Just to increase coverage? If so, I think we can leave it for now until #276 is resolved.

Comment (yolile) [reply tests/test_settings.py]: Mainly because this test as it is now doesnt test anything so, yes, lets keep this open until #276 is resolved

Comment (jpmckinney) [reply tests/test_settings.py]: Right - then let's remove this test file from the PR.

**Pull request merged**